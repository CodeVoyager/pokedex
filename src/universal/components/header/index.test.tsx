import { shallow } from 'enzyme';
import React from 'react';
import { Header } from '.';

describe('Component::ButtonsContainer', () => {
  it('should render correctly', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
