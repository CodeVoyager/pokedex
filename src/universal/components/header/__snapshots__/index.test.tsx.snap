// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component::ButtonsContainer should render correctly 1`] = `
<div
  className="pokemon-header"
>
  <Link
    className="pokemon-header-main"
    title="PokeDex"
    to="/"
  >
    PokeDex
  </Link>
  <br />
  <span
    className="pokemon-header-sub"
  >
    Totally not a rip-off!
  </span>
</div>
`;
